tests/commonlib/utils.py

```diff
@@ -282,3 +282,10 @@ def get_findings(elastic_client, config_timeout, match_type):
         time.sleep(1)
 
     return result
+
+
+def identifier_by_name(case_identifier, eval_resource) -> bool:
```

Comment:
@oren-zohar I'm agree with you. The [task](https://github.com/elastic/security-team/issues/6115) for this issue already exists. In current context it will be good enough to move out the identifier function from the test function body in order to not duplicate the code.